### DISCUSSION TITLE
cant download the image ai

### DISCUSSION THREAD

**Discussion by lowiq00** [lllyasviel/Fooocus#3560] [Q&A] [answered]
I have tried to download the file over 10 times, but the download often gets stuck between 80% and 90% when using IDM (Internet Download Manager). When I try to resume the download, IDM asks me to enter my GitHub username and password, but that doesn't work. If I restart the download, it gets stuck again. Sometimes the internet connection is disconnected, and I'm not sure why this is happening. Please help me resolve this issue.

**Comment by mashb1t** [comment, marked as answer]
duplicated by https://github.com/lllyasviel/Fooocus/discussions/3129#discussioncomment-10451579
